### PR TITLE
Add localization support

### PR DESCRIPTION
**Pull request by brianclio**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    rails_column_enumerator (1.2.0)
+    rails_column_enumerator (1.2.1)
```
Comment (brianclio) [inline Gemfile.lock]: does this change warrant a 0.1.0 update instead of a 0.0.1?

Comment (michaelcheung) [inline Gemfile.lock]: I think `1.3.1` makes more sense since you're adding functionality.

File: lib/rails_column_enumerator.rb [hunk 2]
```diff
@@ -9,9 +9,12 @@ def self.get_int(val, enum_values = nil, valid_values = nil)
     val
   end
 
-  def self.get_record(val, enum_values = nil, valid_values = nil)
+  def self.get_record(val, enum_values, valid_values, localization_values, helper_name)
     val = get_int(val, enum_values, valid_values)
-    [valid_values[val].to_s.titleize, val]
+    key = valid_values[val]
+    puts localization_values.class
```
Comment (michaelcheung) [inline lib/rails_column_enumerator.rb]: I assume you meant to remove this

File: spec/rails_column_enumerator_spec.rb [hunk 3]
```diff
@@ -18,6 +18,14 @@
     expect(TestClass::FIRST_TWO).to eq([["First", 1], ["Next", 2]])
   end
 
+  it 'usees localizations for helper consts when provided' do
```
Comment (michaelcheung) [inline spec/rails_column_enumerator_spec.rb]: `uses`

File: Gemfile.lock (hunk 1 above)
Comment (brianclio) [inline Gemfile.lock]: That's what I thought

File: lib/rails_column_enumerator.rb (hunk 2 above)
Comment (brianclio) [inline lib/rails_column_enumerator.rb]: derp lol - thanks

File: spec/rails_column_enumerator_spec.rb (hunk 3 above)
Comment (brianclio) [inline spec/rails_column_enumerator_spec.rb]: /facepalm

**Pull request merged**